portal-ui/src/screens/Console/Users/DeleteUser.tsx

```diff
@@ -0,0 +1,159 @@
+        }
+    });
+
+interface IDeleteUserProps {
+    classes: any;
+    closeDeleteModalAndRefresh: (refresh: boolean) => void;
+    deleteOpen: boolean;
+    selectedUser: User | null;
+}
+
+interface IDeleteUserState {
+    deleteLoading: boolean;
+    deleteError: string;
+}
+
+class DeleteUser extends React.Component<
+    IDeleteUserProps,
+    IDeleteUserState
+    > {
+    state: IDeleteUserState = {
+        deleteLoading: false,
+        deleteError: ""
+    };
+
+    removeRecord() {
+        const { deleteLoading } = this.state;
+        const { selectedUser } = this.props;
+        if (deleteLoading) {
+            return;
+        }
+        if (selectedUser == null) {
+            return;
+        }
+        this.setState({ deleteLoading: true }, () => {
+            api
+                .invoke("DELETE", `/api/v1/users/${selectedUser.id}`, {
+                    id: selectedUser.id
+                })
+                .then((res: UsersList) => {
+                    this.setState(
+                        {
+                            deleteLoading: false,
+                            deleteError: ""
+                        },
+                        () => {
+                            this.props.closeDeleteModalAndRefresh(true);
+                        }
+                    );
+                })
+                .catch(err => {
+                    this.setState({
+                        deleteLoading: false,
+                        deleteError: err
+                    });
+                });
+        });
+    }
+
+    render() {
+        const { classes, deleteOpen, selectedUser } = this.props;
+        const { deleteLoading, deleteError } = this.state;
+
+        if (selectedUser === null) {
+            return <div />;
+        }
+
+        return (
+            <Dialog
+                open={deleteOpen}
+                onClose={() => {
+                    this.setState({ deleteError: "" }, () => {
+                        this.props.closeDeleteModalAndRefresh(false);
+                    });
+                }}
+                aria-labelledby="alert-dialog-title"
+                aria-describedby="alert-dialog-description"
+            >
+                <DialogTitle id="alert-dialog-title">Delete User</DialogTitle>
+                <DialogContent>
+                    {deleteLoading && <LinearProgress />}
+                    <DialogContentText id="alert-dialog-description">
+                        Are you sure you want to delete user{" "}<b>{selectedUser.name}</b>?
+                        {deleteError !== "" && (
+                            <React.Fragment>
+                                <br />
+                                <Typography
+                                    component="p"
+                                    variant="body1"
+                                    className={classes.errorBlock}
+                                >
+                                    {`${deleteError}`}
```

Comment:
please change `{`${deleteError}`}` to `{deleteError}`

Reply:
done